docs/releases/5.2.txt

```diff
@@ -487,3 +487,8 @@ Miscellaneous
 * The fallback to ``request.user`` when ``user`` is ``None`` in
   ``django.contrib.auth.login()`` and ``django.contrib.auth.alogin()`` will be
   removed.
+
+Features
+========
```

Comment:
This isn't a new feature. We should move it into `Backwards incompatible changes in 5.2`.

I'd introduce a subsection:

```rst
Templates
---------
```